src/MIDebugEngine/AD7.Impl/AD7StackFrame.cs

```diff
@@ -249,31 +244,29 @@ private void EnsureLocalsAndParameters()
                     await Engine.UpdateRadixAsync(radix);
                 });
             }
-            if (_textPosition != null && (!_hasGottenLocalsAndParams || radix != _radix))
+            //if (_textPosition != null && radix != _radix)
```

Comment:
>if [](http://example.com/codeflow?start=14&length=2)

Eliminating the `radix` check is definitely what you want since `_textPosition != null && (!_hasGottenLocalsAndParams || radix != _radix)` would evaluate to `_textPosition != null && (!false || radix != _radix))` which just turns into `_textPosition != null`.

I am assuming the `_textPosition != null` check is there to avoid trying to fetch locals from a frame without symbols. I am not 100% sure if that check is correct, but I think I would put that part of the check back since I can see why that might be correct.